supertokens_python/recipe/session/recipe_implementation.py

```diff
@@ -147,27 +159,57 @@ def update_jwt_signing_public_key_info(
     async def create_new_session(
         self,
         request: BaseRequest,
+        response: BaseResponse,
         user_id: str,
         access_token_payload: Union[None, Dict[str, Any]],
         session_data: Union[None, Dict[str, Any]],
         user_context: Dict[str, Any],
     ) -> SessionContainer:
+        log_debug_message("createNewSession: Started")
+        output_transfer_method = self.config.get_token_transfer_method(
+            request, True, user_context
+        )
+        if output_transfer_method == "any":
+            output_transfer_method = "header"
+
+        log_debug_message(
+            "createNewSession: using transfer method %s", output_transfer_method
+        )
+
+        disable_anti_csrf = output_transfer_method == "header"
+
         session = await session_functions.create_new_session(
-            self, user_id, access_token_payload, session_data
+            self,
+            user_id,
+            disable_anti_csrf,
+            access_token_payload,
+            session_data,
         )
+
+        transfer_methods: List[TokenTransferMethod] = available_token_transfer_methods  # type: ignore
```

Comment:
why type ignore?

Reply:
It was because of some issues with Literal. Have figured out a cleaner way for this now. Directly using `available_token_transfer_methods` now. No need of `transfer_methods`